ccdproc/tests/test_ccdmask.py

```diff
@@ -0,0 +1,192 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# This module implements the base CCDData class.
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from numpy.testing import assert_array_equal
+
+import numpy as np
+
+from ..core import ccdmask
+from ..ccddata import CCDData
+
+def test_ccdmask_pixels():
+    flat1 = CCDData(data=np.array([[
```

Comment:
This fails because it's not possible to provide the `data` as keyword argument to the `NDDataArray` constructor. Just remove the `data=` and just pass in the array.